docs/src/pages/guides/typescript.md

```diff
@@ -106,5 +106,12 @@ declare module 'material-ui/styles/createMuiTheme' {
       breakpoint: Breakpoint
     }
   }
+  // allow configuration using `createMuiTheme`
+  interface Theme {
+    appDrawer?: {
+      width?: React.CSSProperties['width']
+      breakpoint?: Breakpoint
+    }
+  }
```

Comment:
I don't understand what's going on here... you're duplicating the above `Theme` declaration but with optional properties?

Reply:
Oops, copy paste error. Good thing we have reviews!